src/client.cpp

```diff
@@ -65,6 +65,19 @@ RawVector fetch(CharacterVector server, CharacterVector method, RawVector reques
   // perform request
   grpc_op ops[6];
   grpc_op *op;
+
+  int METADATA_LENGTH = metadata.length() / 2;
```

Comment:
We can check for null here also.

No need to capslock either.

Reply:
I'll remove the capslock, but I can't for the life of me seem to figure out how to check for null...